.babelrc.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Will `babelify`/`babel` find this file? I thought it only looked for `babel.config.js`, `.babelrc`, and `.babelrc.js` by default, with the first one being their recommendation. I know you can specify a custom configuration file, but I don't see that being done here.

Reply:
My bad, fixed. Reference: https://babeljs.io/docs/en/config-files#file-relative-configuration